client/src/components/CommitList.jsx

```diff
@@ -14,7 +14,10 @@ const CommitList = props => {
   const commitsWithMilestoneCommitMessage = commits.filter(commit =>
     milestoneCommits.includes(commit.trim().toLowerCase())
   );
-  const uniqueMilestoneCommits = [...new Set(commitsWithMilestoneCommitMessage)];
+
+  const uniqueMilestoneCommits = [
+    ...new Set(commitsWithMilestoneCommitMessage.map(message => message.toLowerCase()))
```

Comment:
We are transforming the messages `toLowerCase()` in too many places. Probably this can be refactored but might need a lot more testing. We should consider refactoring when we add some unit tests

Reply:
Line 19 looks problematic to me.  In particular the toLowerCase() execution.  You have already dropped them to lower case on line 15.

I think this is the right time to do that refactoring because I agree with you and there are still some code smells in this PR wrt overuse of toLowerCase that ought to be examined more carefully.